src/Amazon.AspNetCore.Identity.Cognito/CognitoSigninManager.cs

```diff
@@ -62,6 +64,34 @@ public class CognitoSignInManager<TUser> : SignInManager<TUser> where TUser : Co
 
             _contextAccessor = contextAccessor ?? throw new ArgumentNullException(nameof(contextAccessor));
         }
+#endif
+#if NETSTANDARD_2_0
+        public CognitoSignInManager(UserManager<TUser> userManager,
+        IHttpContextAccessor contextAccessor,
+        IUserClaimsPrincipalFactory<TUser> claimsFactory,
+        IOptions<IdentityOptions> optionsAccessor,
+        ILogger<SignInManager<TUser>> logger,
+        IAuthenticationSchemeProvider schemes) : base(userManager, contextAccessor, claimsFactory, optionsAccessor, logger, schemes)
+        {
+
+            if (userManager == null)
+                throw new ArgumentNullException(nameof(userManager));
+            if (claimsFactory == null)
+                throw new ArgumentNullException(nameof(claimsFactory));
+
+            if (userManager is CognitoUserManager<TUser>)
+                _userManager = userManager as CognitoUserManager<TUser>;
```

Comment:
(minor), This constructor does not need to be super-efficient, so using is to check the type is fine. But since we already know the case will succeed, let's just do a hard cast and not an as.

Reply:
Agreed, created https://github.com/aws/aws-aspnet-cognito-identity-provider/issues/120

I will proceed and merge the changes to unblock customers.